build.sbt

```diff
@@ -15,7 +15,7 @@ import munit.sbtmunit.BuildInfo.munitVersion
 import sbtcrossproject.CrossPlugin.autoImport.crossProject
 
 lazy val LanguageVersions = LatestScalaVersions
-lazy val LanguageVersion = EarliestScala213
+lazy val LanguageVersion = LatestScala213
```

Comment:
this is restored from the previous change? who is using it, btw?

do we even need LanguageVersion(s)? should we explicitly refer to latest or earliest everywhere?

Reply:
I inlined them, since this is really the default value and one use in 2 test/bench projects